multimedia/M2VDownsizer/Files/Makefile

```diff
@@ -0,0 +1,257 @@
+# Makefile for M2VDowsizer
+#
+# Author:   Robert Kennedy
+# EMAIL:    amtor@hotmail.com
+#
+# License for Makefile:  GPL-2
+#
+# 'make depend' uses makedepend to automatically generate dependencies 
+#               (dependencies are added to end of Makefile)
+# 'make'        build executable file 'M2VDownsizer'
+# 'make clean'  removes all .o and executable files
+#
+#  Note:    There is NO "make install"
+#           After running "make", just copy M2VDownsizer into the desired directory
+#           e.g.    sudo chmod +x ./M2VDownsizer
+#                   sudo cp M2VDownsizer /usr/local/bin
```

Comment:
In the amount of space it took to write this comment, an install target could probably have been written. 😃

Reply:
Yes.  You are right! :-)

Reply:
This has been resolved in the my last push.  I removed the comment and added a `make install` target to the Makefile.